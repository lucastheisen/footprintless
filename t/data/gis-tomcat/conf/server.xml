<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE jndi-resources [
  <!ENTITY jndi-resources SYSTEM "jndi-resources.xml">
]>

<!--
  Licensed to the Apache Software Foundation (ASF) under one or more
  contributor license agreements.  See the NOTICE file distributed with
  this work for additional information regarding copyright ownership.
  The ASF licenses this file to You under the Apache License, Version 2.0
  (the "License"); you may not use this file except in compliance with
  the License.  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
-->

<Server port="8505" shutdown="Wsq8fVD3jz84yyQ3kCkumv3wbw5UVB">
  <!--APR library loader. Documentation at /docs/apr.html -->
  <Listener SSLEngine="on" className="org.apache.catalina.core.AprLifecycleListener"/>
  <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
  <Listener className="org.apache.catalina.core.JasperListener"/>
  <!-- Prevent memory leaks due to use of particular java/javax APIs-->
  <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener"/>
  <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener"/>
  <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener"/>

  <GlobalNamingResources>
    &jndi-resources;
  </GlobalNamingResources>

  <Service name="Catalina">
    <Connector port="8509" protocol="AJP/1.3" redirectPort="8510" connectionTimeout="20000" />

    <Engine defaultHost="localhost" name="Catalina">
      <Host appBase="webapps" autoDeploy="true" name="localhost" unpackWARs="true" copyXML="true" deployXML="true">
        <Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs" pattern="%h %l %u %t &quot;%r&quot; %s %b" prefix="localhost_access_log." resolveHosts="false" suffix=".txt"/>
      </Host>
    </Engine>
  </Service>
</Server>
