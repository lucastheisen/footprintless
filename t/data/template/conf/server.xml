<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE jndi-resources [
  <!ENTITY jndi-resources SYSTEM "jndi-resources.xml">
]>

<!-- GENERATED BY T, DO NOT EDIT -->

<Server port="${T{foo.tomcat.shutdown.port:26005}}" shutdown="${T{foo.tomcat.shutdown.password:SHUTDOWN}}">
  <!--APR library loader. Documentation at /docs/apr.html -->
  <Listener SSLEngine="on" className="org.apache.catalina.core.AprLifecycleListener"/>
  <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
  <Listener className="org.apache.catalina.core.JasperListener"/>
  <!-- Prevent memory leaks due to use of particular java/javax APIs-->
  <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener"/>
  <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener"/>
  <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener"/>

  <GlobalNamingResources>
    &jndi-resources;
  </GlobalNamingResources>

  <Service name="Catalina">${T_perl{
$port = property('foo.tomcat.http.port');
$protocol = default('foo.tomcat.http.protocol:HTTP/1.1');
$redirect_port = property('foo.tomcat.http.redirectPort');
$connection_timeout = default('foo.tomcat.http.connectionTimeout:20000');
$port
? "\n    <Connector port=\"$port\"" .
  " protocol=\"$protocol\"" .
  ($redirectPort ? " redirectPort=\"$redirect_port\"" : '') .
  " connectionTimeout=\"$connection_timeout\" />"
: ''}}${T_perl{
$port = property('foo.tomcat.ajp.port');
$protocol = default('foo.tomcat.ajp.protocol:AJP/1.3');
$redirect_port = property('foo.tomcat.ajp.redirectPort');
$connection_timeout = default('foo.tomcat.ajp.connectionTimeout:20000');
$port
? "\n    <Connector port=\"$port\"" .
  " protocol=\"$protocol\"" .
  ($redirectPort ? " redirectPort=\"$redirect_port\"" : '') .
  " connectionTimeout=\"$connection_timeout\" />"
: ''}}

    <Engine defaultHost="localhost" name="Catalina">
      <Host appBase="webapps" autoDeploy="true" name="localhost" 
        unpackWARs="true" copyXML="true" deployXML="true">
        <Valve className="org.apache.catalina.valves.AccessLogValve" 
          directory="logs" pattern="%h %l %u %t &quot;%r&quot; %s %b" 
          prefix="localhost_access_log." resolveHosts="false" suffix=".txt"/>
      </Host>
    </Engine>
  </Service>
</Server>
