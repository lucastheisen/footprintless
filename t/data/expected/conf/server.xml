<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE jndi-resources [
  <!ENTITY jndi-resources SYSTEM "jndi-resources.xml">
]>

<!-- GENERATED BY T, DO NOT EDIT -->

<Server port="8505" shutdown="password">
  <!--APR library loader. Documentation at /docs/apr.html -->
  <Listener SSLEngine="on" className="org.apache.catalina.core.AprLifecycleListener"/>
  <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
  <Listener className="org.apache.catalina.core.JasperListener"/>
  <!-- Prevent memory leaks due to use of particular java/javax APIs-->
  <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener"/>
  <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener"/>
  <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener"/>

  <GlobalNamingResources>
    &jndi-resources;
  </GlobalNamingResources>

  <Service name="Catalina">
    <Connector port="8580" protocol="HTTP/1.1" connectionTimeout="20000" />
    <Connector port="8509" protocol="AJP/1.3" connectionTimeout="20000" />

    <Engine defaultHost="localhost" name="Catalina">
      <Host appBase="webapps" autoDeploy="true" name="localhost" 
        unpackWARs="true" copyXML="true" deployXML="true">
        <Valve className="org.apache.catalina.valves.AccessLogValve" 
          directory="logs" pattern="%h %l %u %t &quot;%r&quot; %s %b" 
          prefix="localhost_access_log." resolveHosts="false" suffix=".txt"/>
      </Host>
    </Engine>
  </Service>
</Server>
